assets/price-per-item.js

```diff
@@ -0,0 +1,96 @@
+class PricePerItem extends HTMLElement {
+  constructor() {
+    super();
+    this.input = this.querySelector('input');
+    this.variantId = this.querySelector('input[name="quantity"]').dataset.variantId;
```

Comment:
Is this `this.querySelector('input[name="quantity"]')` the same element as `this.input`?

Reply:
Actually I found a bug that was happening when you switched variant because I never actually updated variant ID in my logic. Fixed.